tox.ini

```diff
@@ -67,8 +69,8 @@ commands =
 description = Run integration tests
 deps =
     jinja2
-    git+https://github.com/juju/python-libjuju.git
-    #juju
+    #git+https://github.com/juju/python-libjuju.git
```

Comment:
I would just delete this comment.

Reply:
Agreed

Reply:
Done. We do toggle them quite often though, don't we?